README.md

```diff
@@ -62,10 +62,10 @@ Please see [SECURITY.md](./SECURITY.md).
 
 | CometBFT version | Requirement | Version        | Tested with |
 |------------------|-------------|----------------|-------------|
-| main             | Go version  | 1.21 or higher | 1.21        |
-| v1.x             | Go version  | 1.21 or higher | 1.21        |
-| v0.38.x          | Go version  | 1.20 or higher | 1.21        |
-| v0.37.x          | Go version  | 1.20 or higher | 1.21        |
+| main             | Go version  | 1.22 or higher | 1.22        |
+| v1.x             | Go version  | 1.22 or higher | 1.22        |
+| v0.38.x          | Go version  | 1.20 or higher | 1.22        |
+| v0.37.x          | Go version  | 1.20 or higher | 1.22        |
```

Comment:
My opinion here is that for 


v0.34.x - we should not deprecate this comet at this time because it is the eternal release.  Maybe it gets deprecated at 1.0 + 1 release after.  I really like 34, even though it is super old.  For as long as we will support bugfixes, I'm a proponent of bumping the golang on the 34 release series to the latest release, at all times.  There's a technical reason for this -- when chains upgrade, they should always upgrade to the very latest go.  This prevents a situation where there are mixed golang runtimes. 
 v0.37.x - we should bump it to 1.22
v0.38.x - we should bump it to 1.22

Reply:
My opinion here is that for 


v0.34.x - we should not deprecate this comet at this time because it is the eternal release.  Maybe it gets deprecated at 1.0 + 1 release after.  I really like 34, even though it is super old.  For as long as we will support bugfixes, I'm a proponent of bumping the golang on the 34 release series to the latest release, at all times.  There's a technical reason for this -- when chains upgrade, they should always upgrade to the very latest go.  This prevents a situation where there are mixed golang runtimes. 
 v0.37.x - we should bump it to 1.22
v0.38.x - we should bump it to 1.22

Reply:
Agree with @faddat wrt upgrades.